app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallScreen.kt

```diff
@@ -56,9 +58,10 @@ private fun OngoingCallContent(ongoingCallViewModel: OngoingCallViewModel) {
         sheetContent = {
             with(ongoingCallViewModel) {
                 CallingControls(
-                    callEstablishedState,
-                    { muteOrUnMuteCall() },
-                    { hangUpCall() }
+                    ongoingCallState = callEstablishedState,
+                    onMuteOrUnMuteCall = { muteOrUnMuteCall() },
+                    onHangUpCall = { hangUpCall() },
+                    onToggleVideo = { toggleVideo() }
```

Comment:
probably it cannot do any harm this lambda here, but to be sure it is safer to lcall the methods like this : 

onMuteOrUnMuteCall = viewModel::muteOrUnMuteCall
onHangUpCall = viewmodel::hangUpCall etc.

Reply:
safer from what ?

Reply:
yes but safer from what ?

Reply:
minor thing, but, could you change it to ? 

```
 onMuteOrUnMuteCall = muteOrUnMuteCall,
                    onHangUpCall = hangUpCall,
                    onToggleVideo =  toggleVideo
```

Reply:
The compiler does not accept it

Reply:
ahh I am sorry i meant this 

onMuteOrUnMuteCall = ::muteOrUnMuteCall, ::onHangUpCall = ::hangUpCall, onToggleVideo = ::toggleVideo

Reply:
could I know what is the benefit of using `method reference` instead `lambda` here ?